.env.example

```diff
@@ -13,6 +13,7 @@ DB_PASSWORD='database_password'
 WP_ENV='development'
 WP_HOME='http://example.com'
 WP_SITEURL="${WP_HOME}/wp"
+WP_DEBUG_LOG=/path/to/debug.log
```

Comment:
Default value should be `true` imo so that it goes with default WordPress behavior.